### ISSUE TITLE
Midterm evaluation by adk87

### ISSUE THREAD

**Issue by andrewkozma** [SiyuSiyuYang/ORIE4741_Project#4]
This project's aim is to determine the outcome of the 2016 presidential election in Florida. The method you are using to do this is to track tweets that come from each county. Then you will use feature transformations of those tweets to predict the next president.

I think this is a very interesting idea for a project. I have never heard of any sort of predictions for president coming in this manner. Most of the predictions it seems come from political factors of the current state of the country. I am curious though why you decided to choose tweets from the three weeks leading up to the election. Also what will be your test set and training set when building your model? You do not have access to the tweets from the three weeks leading up to the primaries, which would have been a very good way to verify your model with known results. Do you plan on predicting the next president before the election happens in a couple weeks, or will you build your model based on the results of the election? You mentioned that Twitter allows 1% of all the public tweets to be sampled; do you know what criteria they use to determine which ones go public? Is it random or would not make certain tweets public? (say they had vulgar language).

I like the different feature transformations you are using in your model. I think your "lexicon based sentiment analysis" that was mentioned is a very interesting way of determining the sentiment of the tweet. Is there any way you can tell if a tweet is sarcastic or not? For example: "x will be such a good candidate #Not," could be a sarcastic tweet. 

To avoid overfitting you mentioned that you will perform linear regression on only the first order. I think you can create a more complex model without worrying about overfitting. You mentioned that the linear model didn't produce that useful results, so you probably understand.

I see a problem with using the voting results from in the primary elections to test your model. In the primary election, there were other candidates for the public to choose from. How will your model take into consideration the fact that those candidates are not in this election? Is your model just trying to predict Democrat versus Republican, or will it choose a specific candidate from a list of candidate, in which case it can be useful in the primaries.  

Here are some things you might want to consider if you haven't considered them already. How will you deal with those tweets that don't mention political candidates? People who post political tweets still have a chance to vote in the election, and how will your model decide from which candidate they choose? Do the tweets take into consideration the age of the person who has the account? If there are a bunch of political posts, but the person posting them is under 18, they won't be able to vote and will have no impact on the election. You mentioned you will be using a circle to approximate the counties. Is this a reasonable approximation? Are any of the counties very narrowly shaped? Also, the midterm report mentions that the radius of this circle is the square root of the area, but do you need to take pi (~ 3.14) into account in the formula?
